pom.xml

```diff
@@ -8,7 +8,7 @@
         <version>3.2.5</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
-    <groupId>nl.dtls</groupId>
+    <groupId>org</groupId>
```

Comment:
`org` is not a proper `groupId`, I'd vote for `groupId` set to `org.fairdatapoint` and leave `artifactId` at `fairdatapoint`.